src/Controller/BackendController.php

```diff
@@ -224,8 +226,23 @@ public function alertsAction()
      */
     public function pickerAction(Request $request)
```

Comment:
Can we have some extended description in the docblock here? :) `Handles the picker redirect.` does not really help a lot.